tests/factories/event.py

```diff
@@ -20,4 +20,4 @@ class Meta:
     # Using an Incident/Event factory here fails with a mysterious error:
     # https://github.com/pytest-dev/pytest-django/issues/713 (using @pytest.mark...
     # didn't resolve it). For now, a static fixture suffices.
-    payload = {"report": "we're out of milk", "impact": "making tea is difficult"}
+    payload = {"report": "we are out of milk", "impact": "making tea is difficult"}
```

Comment:
I don't think this is a functional change by the way - it was just making the test output a bit confusing to read :)